hecuba_core/src/TupleRowFactory.cpp

```diff
@@ -206,12 +207,30 @@ int TupleRowFactory::cass_to_c(const CassValue *lhs, void *data, int16_t col) co
             cass_iterator_free(tuple_iterator);
             return 0;
         }
+        case CASS_VALUE_TYPE_DATE: {
+            cass_uint32_t year_month_day;
+            CassError rc = cass_value_get_uint32(lhs, &year_month_day);
+            CHECK_CASS("TupleRowFactory: Cassandra to C parse uint32 unsuccessful, column:" + std::to_string(col));
+            if (rc == CASS_ERROR_LIB_NULL_VALUE) return -1;
+            time_t time = (time_t) cass_date_time_to_epoch(year_month_day, 0);
+            int64_t _time = *((int64_t *) &time);
```

Comment:
This doesn't make sense. You make 3 differents casts, but you only need one:
time = (int64_t) cass_date_time_to_epoch(year_month_day, 0);
Also, the variables never start with underscore in C/C++